rules/BUILD

```diff
@@ -31,3 +32,35 @@ pkg_tar(
         "cc-sample-project/test.cc",
     ],
 )
+
+skylark_library(
```

Comment:
Does this affect how environments.bzl is used? Before we were calling 
load("@bazel_toolchains//rules:environments.bzl", "clang_env")

is it now

load("@bazel_toolchains//rules:environment", "clang_env")?

Reply:
No it does not affect how load statements are used, I think it affects mainly how the files are used to generate all the documentation for a set of skylark rules (https://skydoc.bazel.build/docs/getting_started.html) - which we should also do soon, and for tests - https://github.com/bazelbuild/bazel-skylib#bzl_library (though I honestly don't know much about tests that require use of this, but will find out more if I can).

Reply:
Ack.